src/services/codefixes/inferFromUsage.ts

```diff
@@ -731,12 +760,16 @@ namespace ts.codefix {
 
         function inferFromContext(usageContext: UsageContext, checker: TypeChecker) {
             const types = [];
+
             if (usageContext.isNumber) {
                 types.push(checker.getNumberType());
             }
-            if (usageContext.isString) {
+            else if (usageContext.isString) {

```

Comment:
In the case that we have an unambiguous number usage, *and* an unambiguous string usage, now we'll always prefer `number`?

Reply:
Oops, yeah, that's wrong. Should be just `if` instead of `else if`, and unifyFromContext will handle the resulting type set correctly.